model/project.go

```diff
@@ -1454,14 +1454,23 @@ func (p *Project) ResolvePatchVTs(patchDoc *patch.Patch, requester, alias string
 
 	if alias != "" {
 		catcher := grip.NewBasicCatcher()
-		vars, err := FindAliasInProjectOrRepo(p.Identifier, alias)
-		catcher.Add(errors.Wrap(err, "can't get alias from project"))
-		if len(vars) == 0 && len(patchDoc.PatchedProjectConfig) > 0 {
-			projectConfig, err := createProjectConfig([]byte(patchDoc.PatchedProjectConfig))
-			catcher.Add(errors.Wrap(err, "can't retrieve aliases from patched config"))
-			if err == nil {
-				projectConfigAliases := aliasesToMap(getFullProjectConfigAliases(projectConfig))
-				vars = projectConfigAliases[alias]
+		vars, shouldExit, err := FindAliasInProjectOrRepoFromDb(p.Identifier, alias)
+		catcher.Wrap(err, "can't get alias from project")
+		if !shouldExit && len(vars) == 0 {
+			if len(patchDoc.PatchedProjectConfig) > 0 {
+				projectConfig, err := createProjectConfig([]byte(patchDoc.PatchedProjectConfig))
+				catcher.Wrap(err, "can't retrieve aliases from patched config")
+				if err == nil {
+					vars, err = findAliasFromProjectConfig(projectConfig, alias)
+					if err != nil {
```

Comment:
Similar to above, we don't need to check if err is nil, since catcher.Wrapf will noop if there is no error (same with 1472)